mne/decoding/base.py

```diff
@@ -0,0 +1,174 @@
+"""Base class copy from sklearn.base"""
+# Authors: Gael Varoquaux <gael.varoquaux@normalesup.org>
+#          Romain Trachel <trachelr@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import inspect
+import warnings
+import six
+import numpy as np
+
+
+class BaseEstimator(object):
```

Comment:
Yep it is a copy-paste from sklearn... @agramfort told me to do this.
I think he wanted to avoid dependencies for base classes